src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -1,6 +1,7 @@
 //
 // Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Licensed under the MIT license. See LICENSE file in the project root for full
+// license information.
```

Comment:
I don't remember the copyright not fitting in corefx. Do we have the same max width?